recipes/earlgrey/meta.yaml

```
@@ -11,7 +11,8 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 3
```

```suggestion
  number: 2
```
Since build 2 never got to master or the channel, you don't need to bump it twice within the same PR.